taccsite_cms/templates/guides/globus_data_transfer.html

```diff
@@ -0,0 +1,264 @@
+      </p>
+
+      <p>
+        Go to
+        <a target="_blank" href="https://globus.org/">https://globus.org</a>
+        and log in.
+      </p>
+
+      <img src="https://a2cps-staging.tacc.utexas.edu/media/filer_public_thumbnails/filer_public/64/ec/64ec67d5-29fb-4a8c-84bf-d8e52860083a/picture4.png__1170x0_q85_subsampling-2_upscale.png" />
+
+      <p>
+        Upon successful login you, you will be directed to the "File Manager" landing page.
+      </p>
+
+      <img src="https://a2cps-staging.tacc.utexas.edu/media/filer_public_thumbnails/filer_public/04/cc/04cc029b-9af8-4cb0-8698-943ae93d9e0e/picture5.png__1170x0_q85_subsampling-2_upscale.png" />
+
+      <p>Click on Endpoints.</p>
+
+      <img src="https://a2cps-staging.tacc.utexas.edu/media/filer_public_thumbnails/filer_public/14/a5/14a540bd-9666-449c-be18-ef18456faf42/picture6.png__1170x0_q85_subsampling-2_upscale.png" />
+
+      <p>
+        Click “+ Create new endpoint” and follow the instructions to set up your desktop/laptop as an endpoint.
+      </p>
+
+      <img src="https://a2cps-staging.tacc.utexas.edu/media/filer_public_thumbnails/filer_public/71/94/71947964-5a11-4bb3-ab5e-b8b2ae1b1c86/picture7.png__1170x0_q85_subsampling-2_upscale.png" />
+
+      <p>
+        Enter a Display Name to identify your local endpoint like My Laptop, My Desktop at Home, etcetera and then click Generate Setup Key and click copy to copy the Personal Setup Key.
+      </p>
+
+      <p>Download and Install the Globus Connect Personal client.</p>
+
+      <p>
+        After install, open the Globus Connect Personal application. A pop menu pops up asking your setup key. Copy the setup key from the previous step to complete the setup.
+      </p>
+
+      <p>
+        Click on “File Manager”, and next click on the Collection field. You can choose "Your collections" and click on "My Laptop" to select the created endpoint to your computer.
+      </p>
+
+      <img src="https://a2cps-staging.tacc.utexas.edu/media/filer_public_thumbnails/filer_public/6f/ee/6feecdb0-0abc-4732-85a0-88f3e58a361f/picture8.png__1170x0_q85_subsampling-2_upscale.png" />
+
+      <p>You can now access the files on your desktop/laptop via Globus.</p>
+
+      <img src="https://a2cps-staging.tacc.utexas.edu/media/filer_public_thumbnails/filer_public/0f/3d/0f3daa28-6632-4d3e-a964-706d33dc2fa3/picture9.png__1170x0_q85_subsampling-2_upscale.png" />
+
+      <p>
+        You can also click on Panels to look at two endpoints at the same time. In the other transfer endpoint, search for "TACC" and select the appropriate allocation storage system (Frontera, Stampede2, Corral, Ranch, etcetera) for the desired data.
+      </p>
+      <details class="small"><summary>Examples:</summary>
+        <dl>
+          <dt>UTRC Portal</dt>
```

Comment:
And here

Reply:
Please elaborate.

This line is an identifier for example values for UTRC Portal. Beneath it, after the values, is the same content for Frontera Portal.

Reply:
My mistake, this is part of example text.